install/install-by-msi.md

```diff
@@ -45,157 +41,67 @@ simplest example to output any incoming records to td-agent's log file.
 </match>
 ```
 
-After you've installed .msi package, you'll see the program called
-`Td-agent Command Prompt` installed. Please double click this icon in
-the Windows menu (below is how it looks like on Windows Server 2012).
+A new program `Td-agent Command Prompt` is installed as part of `td-agent`.
+Open this Command Prompt from the Windows Start menu.
+
+Its icon looks like this on Windows Server 2012:
 
-![](/images/msi-td-agent-command-prompt.png)
+![msi-td-agent-command-prompt.png](/images/msi-td-agent-command-prompt.png)
 
-In the prompt, please execute the command below to launch td-agent
-process.
+Now, launch `td-agent` with the following command:
 
 ```
 > fluentd -c etc\td-agent\td-agent.conf
 ```
 
-Then, please launch another `Td-agent Command Prompt` and type the
-command below. This will send a record to td-agent process.
+Then, open another `Td-agent Command Prompt` instance and type the following
+command to send a record to `td-agent`:
 
 ```
 > echo {"message":"hello"} | fluent-cat test.event
 ```
 
-It's working properly if td-agent process outputs the message
```

Comment:
This reverted https://github.com/fluent/fluentd-docs-gitbook/pull/190 commit. Keep PR190 changes.